src/App.js

```diff
@@ -35,6 +35,13 @@ class App extends Component {
       queryType = queryType ? queryType.toLowerCase() : 'sql'
     }
 
+    // hack to get the large dataset `covid_19_cases` to load
+    if (query === 'SELECT *\nFROM covid_19_cases') {
+      const fiveDaysAgo = new Date()
+      fiveDaysAgo.setDate(new Date().getDate() - 5)
+      query = `SELECT *\nFROM covid_19_cases\nWHERE date > ${fiveDaysAgo.toISOString().slice(0, 10)}`
+    }
+
     if (parsedQueryString.state) {
       ({dataset_name, query, queryType, addQuery, forceTableau} = getStateObject(parsedQueryString.state))
```

Comment:
Hm maybe I should overwrite it after we pull it out of state though...?

Reply:
Can we do last 4 weeks?

Reply:
oh sure not sure where I got 5 days from anyway